example_app_generator/generate_app.rb

```diff
@@ -28,6 +28,10 @@
     gsub_file "Gemfile", /.*rdoc.*/, "gem 'rdoc', '< 6'"
   end
 
+  if Rails::VERSION::STRING >= '6'
+    gsub_file "Gemfile", /.*jbuilder.*/, "gem 'jbuilder', :git => 'https://github.com/rails/jbuilder.git', :branch => 'master'"
```

Comment:
I would prefer to stick to a commit instead of living on the edge :)

What do you think? \cc @JonRowe

Reply:
For pre release I think it's ok

Reply:
Thanks Jon